### PR TITLE
Implement the basic GPIO voltage level collection

### PR DESCRIPTION
**Pull request by hbgxsm95**
In this PR, I wrote the documentation for the environment configuration and wrote a simple script to transfer the binary executable file from the workstation to the raspberry pi and run it.

The main.go collects the voltage on the GPIO with the index of 11 and controls the LED on the GPIO 7.

### CODE REVIEW

File: go/iot/main.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package main
+
+// Copyright 2017, OpenCensus Authors
```
Comment (songy23) [inline go/iot/main.go]: nit: s/2017/2018

File: go/iot/main.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+import (
+	"github.com/mgutz/logxi/v1"
+	"gobot.io/x/gobot"
+	"gobot.io/x/gobot/drivers/gpio"
+	"gobot.io/x/gobot/platforms/raspi"
+	"time"
+)
+
+// Copyright 2017, OpenCensus Authors
```
Comment (songy23) [inline go/iot/main.go]: Another nit: the license should be moved to the top line.

Comment (hbgxsm95) [reply go/iot/main.go]: Okay.

**Comment by hbgxsm95** [comment]
@rakyll Hi Jaana, I would really appreciate it you could review the simple starter codes in your spare time~

File: go/iot/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+### Installation 
+
+`cd $GOPATH` 
+
+`go get -u github.com/census-ecosystem/opencensus-experiments`
```
[the remark this answers is not in the document]
Comment (hbgxsm95) [reply go/iot/README.md]: But we still need to run the 'go get -u bulabulbulabula' before we cd to that directory I think?

**Pull request merged**